app/layout.tsx

```diff
@@ -1,18 +1,19 @@
 import type { Metadata } from "next";
 import { Poppins } from "next/font/google";
 import '@fortawesome/fontawesome-svg-core/styles.css'
+import "@/styles/globals.scss";
 import Header from "@/components/Header";
 import Footer from "@/components/footer";
 import { ApolloWrapper } from "@/lib/graphql/apollo-wrapper";
 
-import "./globals.scss";
-
 
+// TODO: Why this method? Is the CSS Import not enough in globals.css?
```

Comment:
L10: @andrewebdev - We should be using this NextJS font optimization, so we can leave it here. I believe the import of Poppins in global.scss was just copied over from the dmptool when putting in filler content. That import can be removed.